cypress/integration/pages/articles/testsForCanonicalOnly.js

```diff
@@ -43,6 +46,47 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
       );
     });
 
+    if (
+      serviceHasInlineLink(service) &&
+      (Cypress.env('APP_ENV') === 'local' || Cypress.env('APP_ENV') === 'test')
+    ) {
+      describe('Focus', () => {
+        describe('Navigation between articles from different services', () => {
+          it('should always focus on `h1#content` when user navigates to previously visited articles using the `back` and `forward` button', () => {
+            cy.get('main a[href*="/articles/"]').click();
+            cy.go('back');
+            cy.focused().should('have.id', 'content');
```

Comment:
I wondered if all the assertions that `#content` is focused should be accompanied by some preconditions that `#content` does _not have focus_ beforehand. Otherwise the tests could still pass even though navigation/focus changing is broken.

Reply:
Thanks @jamesdonoh . This has been added